### PR TITLE
Amir / Fix: enable create account button after selecting a currency

### PR DESCRIPTION
**Pull request by amirghadimi-fs**
Expected behavior:
After we choose currency the Create account button should go back to enable.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6372,14 +6355,12 @@
...
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "yallist": {
           "version": "3.0.3",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```
Comment (negarn) [inline package-lock.json:6363]: why are there changes to package-lock in this PR?

File: src/javascript/app/pages/user/set_currency.js [hunk 2]
```diff
@@ -152,6 +152,10 @@ const SetCurrency = (() => {
             const $clicked_currency = $(this);
             $currency_list.find('> div').removeClass('selected');
             $clicked_currency.addClass('selected');
+            
+            if ($submit) {
+                $submit.removeClass('button-disabled');
```
Comment (negarn) [inline src/javascript/app/pages/user/set_currency.js:157]: we shouldn't be doing this here, this is on line 275 right before showing the error, so we should figure out why that line isn't working and fix that instead

File: package-lock.json (hunk 1 above)
Comment (amirghadimi-fs) [reply package-lock.json:6363]: It seams I used a different version of "npm". I'll fix it.

File: src/javascript/app/pages/user/set_currency.js [hunk 3]
```diff
@@ -276,13 +276,27 @@ const SetCurrency = (() => {
...
     };
 
+    /**
+     * Remove error text if $error is defined
+     * Enable confirm button if isBtnEnable is true
+     *
+     * @param {object} $error // error text jquery element
+     * @param {boolean} isBtnEnable // Enable button
+     */
+    const removeError = ($error, isBtnEnable)=>{
```
Comment (negarn) [inline src/javascript/app/pages/user/set_currency.js:291]: for variables, we separate them with underscores:
```suggestion
    const removeError = ($error, is_btn_enabled) => {
```

**Review by negarn — APPROVED** [review]

File: src/javascript/app/pages/user/set_currency.js [hunk 4]
```diff
@@ -276,13 +276,27 @@ const SetCurrency = (() => {
...
     };
 
+    /**
+     * Remove error text if $error is defined
+     * Enable confirm button if is_btn_enabled is true
+     *
+     * @param {object} $error // error text jquery element
+     * @param {boolean} is_btn_enabled // Enable button
+     */
+    const removeError = ($error, is_btn_enabled)=>{
```
Comment (negarn) [inline src/javascript/app/pages/user/set_currency.js:291]: please add the same spacing that we generally do
```suggestion
    const removeError = ($error, is_btn_enabled) => {
```

**Review by negarn — APPROVED** [review]

**Pull request merged**